modules/migration-excluding-resources.adoc

```diff
@@ -0,0 +1,150 @@
+
+[id='excluding-images-from-migration-plan_{context}']
+== Excluding imagestreams
+
+You can exclude imagestreams from a migration by disabling imagestream migration in the Migration Controller CR.
+
+.Procedure
+
+. Edit the Migration Controller CR:
++
+----
+$ oc edit migrationcontroller -n openshift-migration
+----
+
+. Add `disable_image_migration: true` to the `spec` section:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  disable_image_migration: true
+...
+----
+
+. Wait two minutes for the Migration Controller Pod to restart so that the changes are applied.
+
+. Verify that imagestream migration is disabled by viewing the excluded resources:
++
+----
+$ oc get deployment -n openshift-migration migration-controller -o yaml | grep EXCLUDED_RESOURCES -A1
+----
++
+`imagestreams` is displayed among the values of the `EXCLUDED_RESOURCES` parameter:
++
+[source,yaml]
+----
+    - name: EXCLUDED_RESOURCES
+      value:
+      imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,imagestreams
+----
+
+[id='excluding-pvs-from-migration-plan_{context}']
+== Excluding persistent volumes from a migration
+
+You can exclude persistent volumes (PVs) from a migration by disabling PV migration in the Migration Controller CR. Disabling PV migration also disables PV discovery when you create the migration plan.
+
+.Procedure
+
+. Edit the Migration Controller CR:
++
+----
+$ oc edit migrationcontroller -n openshift-migration
+----
+
+. Add `disable_pv_migration: true` to the `spec` section:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  disable_pv_migration: true
+...
+----
+
+. Wait two minutes for the Migration Controller Pod to restart so that the changes are applied.
+
+. Verify that PV migration is disabled by viewing the excluded resources:
++
+----
+$ oc get deployment -n openshift-migration migration-controller -o yaml | grep EXCLUDED_RESOURCES -A1
+----
++
+`persistentvolumes`, and `persistentvolumeclaims` are displayed among the values of the `EXCLUDED_RESOURCES` parameter:
++
+[source,yaml]
+----
+    - name: EXCLUDED_RESOURCES
+      value:
+      imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,persistentvolumes,persistentvolumeclaims
+----
+
+[id='excluding-other-resources-from-migration-plan_{context}']
+== Excluding other resources from a migration
+
+You can add exclude other resources from a migration by adding them to the `excluded_resources` parameter in the MigrationController CR.
```

Comment:
What other resources can you exclude like this? Do you have a list?

Reply:
No list. Has to be decided on a case-by-case basis because the reason for excluding the resource is up to the user. Sometimes they want to migrate with a different tool or they don't want to migrate a resource for some other reason.